extensions/amp-soundcloud/1.0/converter.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Converts RGBA into HEX format
+ * @param {string} rgba Input string RGBA(r, g, b, a) format
+ * @return {string} Hexadecimal color
+ */
+export const rgba2hex = (rgba) => {
```

Comment:
Since this is only used as a helper in the storybook samples, should we move it into the `storybook/` path?

Reply:
Yes, it should be. I have moved `converter.js` into storybook and updated examples.